src/wallet3/default_daemon_comms.cpp

```diff
@@ -118,7 +117,7 @@ namespace wallet
 
     omq->job([blocks=std::move(blocks),this](){
         for_each_wallet([&](std::shared_ptr<Wallet> wallet){
-            wallet->add_blocks(blocks);
+              wallet->add_blocks(blocks);
```

Comment:
clang-format will probably sort this, but this whitespace change seems incorrect.